Setting up a local instance.md

```diff
@@ -0,0 +1,151 @@
+* Edit the file and fill in the values applicable to your local site where you see empty strings, also the username and password for the Backdrop and CiviCRM database connections.
+
+If you will not be running CiviCRM, you can ignore this.
+
+### `civicrm_table_prefixes.php`
+
+The file `civicrm_table_prefixes.php` contains a list of the CiviCRM database tables that need to be made visible to Backdrop; each entry is prefixed with `civicrm.`, which is the name of the CiviCRM database. If your local instance is changing the name of the CiviCRM database, you will need to change every occurrence here (and then be careful because this file is under version control). 
+
+If you are not running CiviCRM, you can ignore this file.
+
+## Up and running
+
+If you will be running CiviCRM, the site should be fully functional; go to the home page and see if it loads.
+
+### Effects of sanitization
+
+The sanitized `backdropcmsorg` database 
+
+* keeps all user names and other user account fields
+* truncates all cache tables
+* replaces all user account email addresses with a unique but generic value
+* replaces all password hashes with a generic hash
+
+This means, however, that you cannot log into your own website account (because the password has changed), and you cannot get a "reset your password" email link because the email address is no longer valid. However, if the site is functional and you have `bee` installed on your system, you can set your account's password using the `bee` command:
+
+```
+bee upw <username> <password>
+```
+
+The sanitized `civicrm` database
+
+* keeps group membership intact
+* replaces all contact address information with generic values
+* replaces all derived contact names (but not the original name) with unique generic values
+* replaces all emails with unique generic values
+* replaces user account names with unique generic values
+
+### Touch-up
+
+You should disable the `tugboat` and `borg_tugboat` modules so that the local site doesn't try to query Tugboat (and put errors into the dblog). You can do this with `bee` as follows:
+
+```
+bee dis -y borg_tugboat
+bee dis -y tugboat
+```
+
+### Things that won't work on the local site
+
+* Interactions with Tugboat are disabled. If you need to investigate this, in addition to enabling the modules you'll need to enter the `tugboat_access_token` in `settings.local.php`.
+* You can't send out emails without filling in the `smtp_pasword` in `settings.local.php`. (We recommend leaving SMTP disabled unless you really know what you're doing, because we don't want a local site to be sending out random emails—and remember that all user account emails have been sanitized).
+* The `borg_signup_*` values in `settings.local.php` allow a site to send a request to the BackdropCMS.org site to sign someone up for the newsletter (this includes signups directly from BackdropCMS.org). Without the credentials, that functionality won't work.
+* MORE TO COME
+
+## Running without CiviCRM
```

Comment:
Can we move this above the tugboat information?

Reply:
Yes, done.